core/src/test/scala/com/avast/clients/storage/compression/ZstdDecompressOutputStreamTest.scala

```diff
@@ -0,0 +1,60 @@
+package com.avast.clients.storage.compression
+
+import com.avast.scala.hashes.Sha256
+import com.github.luben.zstd.Zstd
+import org.junit.runner.RunWith
+import org.scalatest.FunSuite
+import org.scalatestplus.junit.JUnitRunner
+
+import java.io.ByteArrayOutputStream
+import java.nio.ByteBuffer
+import java.security.MessageDigest
+
+@RunWith(classOf[JUnitRunner])
+class ZstdDecompressOutputStreamTest extends FunSuite {
+  private def computeSha256(data: Array[Byte]): Sha256 = {
+    Sha256(MessageDigest.getInstance("SHA-256").digest(data))
+  }
+
+  private def generateData(size: Int): Array[Byte] = {
+    val builder = Array.newBuilder[Byte]
+    var i = 0
+    while (i < size) {
+      builder += (i % 256).toByte
+      i += 1
+    }
+    builder.result()
```

Comment:
A bit too Java-ish 😂 
```suggestion
    Random.nextBytes(size)
```

Reply:
I need compressible pseudo data. Completely random bytes are usually not compressible.

Reply:
Made it more Scalish.